doc/testing.html

```diff
@@ -586,6 +592,18 @@ <h3 id="pkcs11-tests">PKCS11 Tests</h3>
     JTREG=&quot;JAVA_OPTIONS=-Djdk.test.lib.artifacts.nsslib-linux_aarch64=/path/to/NSS-libs&quot;</code></pre>
 <p>For more notes about the PKCS11 tests, please refer to
 test/jdk/sun/security/pkcs11/README.</p>
+<h3 id="testing-with-external-security-providers">Testing with external
+security providers</h3>
```

Comment:
Avoid the word "external". Change this to "Testing with an alternate security provider".